packages/synapse-react-client/src/SWC.index.ts

```diff
@@ -1,3 +1,10 @@
+/**
+ * SWC.index.ts represents the entry point for the Synapse React Client library as it is published to NPM and
+ * used by SynapseWebClient.
+ *
+ * If you wish to export a component or other object to be used by SynapseWebClient, you should add it to this file.
+ */
```

Comment:
Renamed umd.index.ts to SWC.index.ts and added this to clarify what this file is.